datasetinsights/datasets/base.py

```diff
@@ -1,44 +1,10 @@
-from abc import ABCMeta, abstractmethod
+from abc import abstractmethod
 
 
-class Dataset(metaclass=ABCMeta):
+class Dataset:
     """Abstract base class for datasets.
     """
 
-    @staticmethod
```

Comment:
Why are these removed in this review? Why is ABCMeta of `Dataset`  removed?

Reply:
This was due to the fact that we no longer have the concept of name for the dataset, maybe i have misunderstood something.

Reply:
I'll revert it